administrator/components/com_fabrik/models/fields/fabrikeditor.php

```diff
@@ -68,10 +68,9 @@ protected function getInput()
 
 		$mode      = $this->getAttribute('mode', 'html');
 		$theme     = $this->getAttribute('theme', 'github');
-		$height    = $this->getAttribute('height', '200px');
+		$height    = $this->getAttribute('height', '300px');
 		$maxHeight = $this->getAttribute('max-height', str_ireplace('px', '', $height) * 2 . 'px');
-		$width     = $this->getAttribute('width', '300px');
```

Comment:
Is the setting of width necessary at all?

2015-04-29 22:21 GMT+03:00 troester <notifications@github.com>:

> In administrator/components/com_fabrik/models/fields/fabrikeditor.php
> <https://github.com/Fabrik/fabrik/pull/1471#discussion_r29370712>:
>
> >  		$maxHeight = $this->getAttribute('max-height', str_ireplace('px', '', $height) * 2 . 'px');
> > -		$width     = $this->getAttribute('width', '300px');
>
> $width should be set to 100%
> $width = $this->getAttribute('width', '100%');
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Fabrik/fabrik/pull/1471/files#r29370712>.
>